pkgs/development/libraries/librep/git.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit
+, pkgconfig, autoreconfHook
+, readline, texinfo
+, gdbm, gmp, libffi }:
+
+let
+  rev = "a1f2db721aa5055e90f6a76fde625946340ed8cf";
+in
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  name = "librep-${rev}";
```

Comment:
Don't make revision as a name part. 
Use date of that commit instead. Please use librep-yyyy-mm-dd.
Apply this to all PR, please.